src/lib/environment/mod_test.rs

```diff
@@ -561,6 +562,31 @@ fn setup_env_empty() {
     assert_eq!(value, "setup_env_empty2");
 }
 
+#[test]
+fn setup_cargo_home() {
+    setup_cwd(None);
+
+    assert_eq!(
+        envmnt::get_or_panic("CARGO_MAKE_CARGO_HOME"),
+        home::cargo_home().unwrap().to_str().unwrap()
+    );
+}
+
+#[test]
+fn setup_cargo_home_overwrite() {
+    let path = Path::new("path");
+    envmnt::set("CARGO_HOME", path);
```

Comment:
can you restore this one to existing/none value? it might be the cause of the ci error (didn't validate, but nice to cleanup such changes)

Reply:
Done.